letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -280,6 +289,14 @@ BootstrapRpmCommon() {
     exit 1
   fi
 
+  # checking if certbot is already packaged with the os
+  if [ "$QUIET" != 1 ]; then
+      if rpm -qa | grep -c certbot -gt 0; then
```

Comment:
You don't need the `-gt 0` here. You also don't need `-c`. Just `if rpm -qa | grep certbot; then`.